README.md

```diff
@@ -10,7 +10,48 @@ The Message Format Working Group (MFWG) is tasked with developing an industry st
 - [Goals and Non-Goals](docs/goals.md)
 - [Record of Consensus Decisions](docs/consensus_decisions.md)
 
-## MessageFormat 2 Draft Syntax
+## MessageFormat 2 Technical Preview
+
+The MessageFormat 2 specification was accepted by the CLDR-TC on 2024-02-28
+as a new part of the LDML specification.
+This specification is being released as a "Tech Preview", 
+which means that the stability policy is not in effect and feedback from
+users and implementers might result in changes to the syntax, data model,
+functions, or other normative aspects of MessageFormat 2.
+Such changes are expected to be minor and, to the extent possible,
+to be compatible with what is defined in the Tech Preview.
+
+The MFWG welcomes any and all feedback, including bugs reports, implementation
+reports, success stories, feature requests, requests for clarification, 
+or anything that would be helpful in stabilizing the specification and
+promoting widespread adoption.
+
+The MFWG specifically requests feedback on the following issues:
+- How to perform non-integer exact number selection [#675](https://github.com/unicode-org/message-format-wg/issues/675)
+- Whether `markup` should support additional spaces [#650](https://github.com/unicode-org/message-format-wg/issues/650)
+- Whether "attribute-like" behavior is needed and what form it should take [#642](https://github.com/unicode-org/message-format-wg/issues/642)
+- Whether to relax constraints on complex message start [#610](https://github.com/unicode-org/message-format-wg/issues/610)
+- Whether omitting the `*` variant key should be permitted [#603](https://github.com/unicode-org/message-format-wg/issues/603)
+
+## What is MessageFormat 2?
+
+Software needs to construct messages that incorporate various pieces of information. 
+The complexities of the world's languages make this challenging.
+MessageFormat 2 defines the data model, syntax, processing, and conformance requirements 
+for the next generation of dynamic messages. 
+It is intended for adoption by programming languages, software libraries, and software localization tooling.
+It enables the integration of internationalization APIs (such as date or number formats),
+and grammatical matching (such as plurals or genders). 
+It is extensible, allowing software developers to create formatting
+or message selection logic that add on to the core capabilities.
```

Comment:
```suggestion
It is extensible, allowing software developers to create formatting
or message selection logic that add on to the core capabilities.
Its data model is — as far as we can tell — capable of
representing messages in all existing syntaxes,
enabling gradual adoption by users of other syntaxes.
```

Reply:
I'm reluctant to commit this as written because it doesn't match in tone. Perhaps:

```suggestion
It is extensible, allowing software developers to create formatting
or message selection logic that add on to the core capabilities.
Its data model provides a means of representing existing syntaxes,
thus enabling gradual adoption by users of older formatting systems.
```